### ISSUE TITLE
[Feature] CldVideoPlayer autoplay should respect browser standard prop of autoplay without captilization

### ISSUE THREAD

**Issue by colbyfayock** [colbyfayock/next-cloudinary#319]
# **Feature Request**

## **Is your feature request related to a problem? Please describe.**

The browser standard video tag uses autoplay without capitalization. The CldVideoPlayer should have the same.

## **Describe the solution you'd like**

`autoplay="always"` should result in `autoplayMode="always"` on the `playerOptions` object.

further, we should support the `autoplay` prop being passed as a boolean, so:

```
<CldVideoPlayer autoplay
<CldVideoPlayer autoplay={true}
```

This however shoudl act as if it were the standard autoplay attribute, which would add `autoplay: true` to the `playerOptions` object.

For backwards compatability, `autoPlay` should still be supported, but pass its value to `autoplay`

To sum up:
- `<CldVideoPlayer autoplay` => `playerOptions.autoplay = true`
- `<CldVideoPlayer autoplay={true}` => `playerOptions.autoplay = true`
- `<CldVideoPlayer autoplay={false}` => `playerOptions.autoplay = false`
- `<CldVideoPlayer autoplay="always"` => `playerOptions.autoplayMode = 'always'`
- `<CldVideoPlayer autoplay="never"` => `playerOptions.autoplayMode = 'never'`
- `<CldVideoPlayer autoPlay="always"` => `playerOptions.autoplayMode = 'always'`

Booleans get passed to `autoplay` and strings get passed to `autoplayMode`

**Comment by codingis4noobs2** [comment]
Hello @colbyfayock , I would like to work on this issue

**Comment by colbyfayock** [comment]
all yours @codingis4noobs2 

wanted to give a heads up that tomororw is the last day for Hacktoberfest contributions

if you open a PR with a majority of the work done, with only edits and tweaks being needed but merged later, it will still count!

**Comment by colbyfayock** [comment]
hey @codingis4noobs2 were you still planning on working on this?

**Comment by codingis4noobs2** [comment]
Hi @colbyfayock , currently my final exams are going on, so I rarely get some time
If I don't make a pr within 2-3 days you may assign it to diff. person 🙂

**Comment by colbyfayock** [comment]
Sounds good. Good luck on the exams!

**Comment by DevYemi** [comment]
Hi @colbyfayock, i would like to pick this up if its still open

**Comment by colbyfayock** [comment]
all yours @DevYemi
